kubernetes/helm_charts/secor/values.j2

```diff
@@ -0,0 +1,55 @@
+azure_account: "{{ sunbird_private_storage_account_name }}"
+azure_secret: "{{ sunbird_private_storage_account_key }}"
+azure_container_name: "telemetry-data-store"
+
+namespace: {{ secor_namespace }}
+
+secor_jobs:
+{% for item in secor_job_list %}
+  {{ item }}:
+    consumer_group: "{{ secor_jobs['%s'|format(item)].consumer_group }}"
+    service_name: "{{ secor_jobs['%s'|format(item)].service_name }}"
+    base_path: "{{ secor_jobs['%s'|format(item)].base_path }}"
+    timestamp_key: "{{ secor_jobs['%s'|format(item)].timestamp_key }}"
+    fallback_timestamp_key: "{{ secor_jobs['%s'|format(item)].fallback_timestamp_key }}"
+    topic: "{{ secor_jobs['%s'|format(item)].topic }}"
+    kafka_broker_host: "{{ secor_jobs['%s'|format(item)].kafka_broker_host }}"
+    zookeeper_quorum: "{{ secor_jobs['%s'|format(item)].zookeeper_quorum }}"
+    max_file_size: "{{ secor_jobs['%s'|format(item)].max_file_size }}"
+    max_file_age: "{{ secor_jobs['%s'|format(item)].max_file_age }}"
+    partition_prefix_enabled: "{{ secor_jobs['%s'|format(item)].partition_prefix_enabled }}"
+    partition_prefix_key: "{{ secor_jobs['%s'|format(item)].partition_prefix_key }}"
+    partition_prefix_mapping: "{{ secor_jobs['%s'|format(item)].partition_prefix_mapping }}"
+    message_channel_identifier: "{{ secor_jobs['%s'|format(item)].message_channel_identifier }}"
+    output_file_pattern: "{{ secor_jobs['%s'|format(item)].output_file_pattern }}"
+    message_parser: "{{ secor_jobs['%s'|format(item)].message_parser }}"
+{% endfor %}
+
+image:
+  prefix: {{ image.prefix }}
+  repository: {{ image.repository }}
+  tag: {{ image.tag }}
+  pullPolicy: {{ image.pullPolicy }}
+
+secor:
+  partition:
+    configFile: /opt/secor/config/secor.partition.properties
+    logConfigFile: /opt/secor/config/log4j.docker.properties
+    monitor:
+      logConfigFile: /opt/secor/config/log4j.docker-warn.properties
+    storage:
+      class: standard
```

Comment:
I dont think we have a storage class named standard in azure. https://docs.microsoft.com/en-us/azure/aks/concepts-storage#storage-classes
Also this must have an optional ansible variable if we want to run on another cloud.

Reply:
I will change the storage class to managed-premium as standard is not a supported storage-class type.

Reply:
don't do that. Just remove the filed altogether, we'll use the default(every kubernetes cluster has `default` storage class). Else you can define a helm variable and override that using ansible. Even for that keep the default value as default.